Sources/RichTextKit/Images/Image+RichText.swift

```diff
@@ -53,6 +53,7 @@ public extension Image {
     static let richTextStyleItalic = symbol("italic")
     static let richTextStyleStrikethrough = symbol("strikethrough")
     static let richTextStyleUnderline = symbol("underline")
+    static let richTextKindLink = symbol("link")
```

Comment:
`richTextLink` is enough :) Place this in alpha order and use it as instructed above 👍